src/main/java/ro/secur/auth/controller/PasswordController.java

```diff
@@ -17,16 +31,37 @@ public PasswordController(UserService userService) {
         this.userService = userService;
     }
 
+    @ApiOperation(value = "forgot password", notes = "Forgot password of a registered user")
+    @ApiResponses(value = {
+            @ApiResponse(code = 400, message = "Bad request"),
+            @ApiResponse(code = 403, message = "Not authorized to change password"),
+            @ApiResponse(code = 404, message = "Email not found in the database"),
+            @ApiResponse(code = 500, message = "Internal server error")
+    })
     @PostMapping("/forgotPassword")
     public void forgotPassword(@RequestBody String email) {
         userService.forgotPassword(email);
     }
 
+    @ApiOperation(value = "reset password", notes = "Reset password of a registered user")
+    @ApiResponses(value = {
+            @ApiResponse(code = 400, message = "Bad request"),
+            @ApiResponse(code = 401, message = "Not authenticated"),
+            @ApiResponse(code = 403, message = "Not authorized to reset password"),
+            @ApiResponse(code = 404, message = "User not found"),
+            @ApiResponse(code = 500, message = "Internal server error")
+    })
     @PutMapping("/resetPassword")
-    public void resetPassword(@RequestParam String token, @RequestBody ChangePasswordRequest request) {
+    public void resetPassword(@RequestParam String token, @RequestBody ResetPasswordRequest request) {
         userService.resetUserPassword(token, request);
     }
 
+    @ApiOperation(value = "expired token", notes = "Token expired")
+    @ApiResponses(value = {
+            @ApiResponse(code = 400, message = "Bad request"),
+            @ApiResponse(code = 401, message = "Not authenticated"),
```

Comment:
You don't need to be authenticated for this one -> part of forgot password functionality